cmd/sqlflowserver/main_test.go

```diff
@@ -268,7 +268,7 @@ func TestEnd2EndHive(t *testing.T) {
 	if testDBDriver != "hive" {
 		t.Skip("Skipping hive tests")
 	}
-	dbConnStr = "hive://127.0.0.1:10000/iris"
+	dbConnStr = "hive://127.0.0.1:10000/iris?auth=NOSASL"
```

Comment:
Maybe add more tests to test connection string with out `?auth=NOSASL`?

Reply:
Will add it in the next pr.